cilium/cmd/helpers.go

```diff
@@ -79,3 +82,46 @@ func requireServiceID(cmd *cobra.Command, args []string) {
 		Usagef(cmd, "Empty service id argument")
 	}
 }
+
+//JSONOutput flag to enable json outout
+var JSONOutput bool
+
+//JSONPath flag to enable Json path support on commands
+var JSONPath string
+
+// AddJSONSupport enable --json --jsonpath flags to the command given
+// in the cmd argument.
+func AddJSONSupport(cmd *cobra.Command) {
+	cmd.Flags().BoolVarP(&JSONOutput, "json", "", false, "Json Output")
+	cmd.Flags().StringVarP(&JSONPath, "jsonpath", "", "", "JsonPath output")
+}
+
+// DumpJSON dump the data variable to the stdout as json.
+// If somethings fail, it'll die with exit code 1.
+// If jsonPath is passed, it'll run the json query over data var.
+func DumpJSON(data interface{}, jsonPath string) {
+
+	if len(jsonPath) == 0 {
+		result, err := json.Marshal(data)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Couldn't marshal to json: '%s'\n", err)
+			os.Exit(1)
+		}
+		fmt.Println(string(result))
+		return
+	}
+
+	parser := jsonpath.New("").AllowMissingKeys(true)
+	if err := parser.Parse(jsonPath); err != nil {
+		fmt.Fprintf(os.Stderr, "Couldn't parse jsonpath expression: '%s'\n", err)
+		os.Exit(1)
```

Comment:
I think it would be better to make this function return error. It would also allow testing it.

Reply:
ack